Duckling/Numeral/DE/NumParser.hs

```diff
@@ -0,0 +1,194 @@
+-- Copyright (c) 2016-present, Facebook, Inc.
+-- All rights reserved.
+--
+-- This source code is licensed under the BSD-style license found in the
+-- LICENSE file in the root directory of this source tree.
+{-# LANGUAGE DeriveFunctor #-}
+
+module Duckling.Numeral.DE.NumParser (parseNumeral) where
+
+import Control.Applicative
+import Data.Char
+import Data.Foldable
+import Data.List
+import Data.String
+import Prelude
+
+newtype Parser a = Parser {runParser :: String -> Maybe (a, String)}
```

Comment:
I'm pretty sure this parser type + NumParser stuff already exists elsewhere in the code case, you should import it